nuocd/pylib/pynuoadmin/nuocmd.py

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env python
+# (C) Copyright NuoDB, Inc. 2021  All Rights Reserved.
```

Comment:
This is copying all of pynuoadmin? Why is that necessary, as opposed to just declaring pynuoadmin as a dependency in `requirements.txt`?

Reply:
I wish I could,   the latest pynuoadmin has not been pushed to pypi.  So to get python3 support I must copy until it is pushed.   I would like to remove this but have the dependency on pypi being updated which engineering seems to be incapable of managing.

Reply:
Do you have a timeframe when that will be available.   I will close this and wait,  but I want to get this out soon as there are people waiting for it.

Reply:
I'm working with release engineering to get it published.

Reply:
@dbutson-mdsol : Version 2.2.0 of pynuoadmin with Python 3 support has been published: https://pypi.org/project/pynuoadmin/

You can remove this now.